### DISCUSSION TITLE
Foundry Fundamentals: Section 1(lesson 13), the add network option of metamask has moved

### DISCUSSION THREAD

**Discussion by 5hr1ganesh** [Cyfrin/foundry-full-course-cu#2814] [General]
### The networks option in Metamask has been removed from the settings as of now, and the Add Network option has been moved to the Bottom of Network selection List.

1. Open MetaMask: Click on the MetaMask extension in your browser.

2. Select Network: Click on the select network dropdown at the top of the MetaMask interface.

3. Add Network: Select "Add Network" from the bottom of dropdown menu.

4. Enter Network Details:

Network Name: Anvil Local Testnet

New RPC URL: http://127.0.0.1:8545 (or http://localhost:8545)

Chain ID: 1337

Currency Symbol: ETH

5. Save: Click Save to add the network.

_For more detailed instructions please visit_ : https://support.metamask.io/networks-and-sidechains/managing-networks/how-to-add-a-custom-network-rpc/

Also when adding the **anvil local network**, Metamask may show the following warnings:

<img width="361" alt="image" src="https://github.com/user-attachments/assets/ebc05b04-3e2a-4971-8d1c-425058460cc7">

But if you have entered all the details correctly, you should be able to save it and import the **anvil test accounts**.

**Comment by EngrPips** [comment]
Thanks for this heads-up @5hr1ganesh
